src/pages/home/owners/Owners.tsx

```diff
@@ -42,7 +45,7 @@ const Owners: FunctionComponent = () => {
             query={ query }
             onSearch={ handleSearch }
           />
-          <TableSkeleton />
+          <TableSkeleton cols={ viewportWidth && viewportWidth > 450 ? 3 : 2 } />
```

Comment:
if these are the breakpoints:
```
    values: {
      xs: 0,
      sm: 600,
      md: 900,
      lg: 1060,
      xl: 1536,
    },
```
and on `src/pages/home/owners/OwnersTable.tsx` line 85: `display: { xs: "none", sm: "inline" },`

seems like we want the 450 value to be 600?, also we probably want to use `theme.breakpoints.values.sm` here 
```suggestion
          <TableSkeleton cols={ viewportWidth && viewportWidth > theme.breakpoints.values.sm ? 3 : 2 } />
```

If the change is made here, we should make it in `Library.tsx` as well, let me know your thoughts